lib/error.h

```diff
@@ -53,6 +53,14 @@ inline void error(const char *format, Args &&...args) {
     auto action = context.getDefaultErrorDiagnosticAction();
     context.errorReporter().diagnose(action, nullptr, format, "", std::forward<Args>(args)...);
 }
+
+template <typename... Args>
+inline void error(cstring format, Args &&...args) {
```

Comment:
Is it possible not to use `cstring` in format strings and go with plain literals? Or there are too many and they are constructed dynamically?